src/main/java/bowling/exception/BowlingException.java

```diff
@@ -0,0 +1,22 @@
+package bowling.exception;
+
+public class BowlingException extends Exception {
```

Comment:
Are the methods being created not default (genuinely don't know)? Since all your other exceptions are extending this and also re-declaring, perhaps that isn't necessary? Also as mentioned earlier, you could have this extend `RuntimeException` and be unchecked. Because people rarely like try-catches and throws written everywhere haha. Especially in the lambda/stream api/futures stuff.